### ISSUE TITLE
[Bitwarden Vault] Always fails with error "Failed to load vault items `Vault is locked`"

### ISSUE THREAD

**Issue by LoKy-dev** [raycast/extensions#8390]
### Extension

https://www.raycast.com/jomifepe/bitwarden

### Description

When I use the "Search vault" the master password is requested, as it should be, and I even see the vault status with my correct account.
After inserting the master password i see the little loader in the bottom left that says "Unlocking Vault...", after that I see the se search ui but it is empty, the "Failed to load vault items `Vault is locked`" error is shown and i'm back to the master password view.

I'm 100% sure that the password is correct because i've literally copy-pasted that in both the web vault and raycast and it works.
i'm 100% sure that the api keys are correct, also tested them from the cli

### Steps To Reproduce

I've just added my api keys, I don't know how to reproduce this behaviour, feel free to ask specific questions

**Comment by jomifepe** [comment]
Hey, thanks for reporting. 

I'm not immediately sure what's happening, so I will investigate it

Is it a fresh installation of the extension?

**Comment by tomstolarczuk** [comment]
This happens to me as well.
Might be related to Bitwarden update.

**Comment by tomstolarczuk** [comment]
@jomifepe reinstall seems to have fixed the issue for me.

**Comment by LoKy-dev** [comment]
@jomifepe uninstalling and reinstalling worked for me too.

Closing this issue.

**Comment by jomifepe** [comment]
Great to hear! I'm gonna investigate the issue.

**Comment by jomifepe** [comment]
Could you please inform me of your CLI version?

**Comment by tomstolarczuk** [comment]
> Could you please inform me of your CLI version?
2023.5.0

**Comment by LoKy-dev** [comment]
2023.7.0 in my case

**Comment by mmellado** [comment]
Just reproduced this, I can provide some additional info 

cli version: 2023.12.0

First I installed the extension. At this point, I didn't have the `bw` CLI installed. I installed it using homebrew from within raycast using the `brew` extension.

I entered key and secret, tried to search the vault with the exception and got the "Failed to load vault items" + Empty vault message.

I uninstalled the extension and reinstalled it. Entered cliend id and secret again, tried to search, got prompted for master key. Entered master key and this time, I got an email from Bitwarden about a login happening from my device (i didn't get this email the previous time). However, I continue to get the "Failed to load vault items" error.

Trying to manually trigger a sync doesn't fix it either.

Trying to login using the cli in my terminal using `bw login --apikey` works. Syncing/listing directly with the CLI from my terminal also works
